src/test/groovy/io/openliberty/tools/gradle/TestLooseApplicationWithWarTask.groovy

```diff
@@ -86,15 +88,23 @@ public class TestLooseApplicationWithWarTask extends AbstractIntegrationTest{
 
```

Comment:
Why doesn't this comparison at the above line need to be OS specific for `targetInArchive`?